### PR TITLE
feat: added login and signup pages

### PR DESCRIPTION
**Pull request by hiteshverma27**

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice work

File: Styles/login.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.login-div{
+    align-items: center;
+    overflow-y: auto;
+    min-height: max-content;
+    background: rgba(255,255,255,0.5);
```
Comment (rohanmathur91) [inline Styles/login.css:5]: have a variable for color

**Review by rohanmathur91 — APPROVED** [review]
LGTM

File: Styles/login.css (hunk 1 above)
Comment (hiteshverma27) [reply Styles/login.css:5]: Will do

**Pull request merged**